.ci/docker/common/install_cuda.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-NCCL_VERSION=v2.21.5-1
+NCCL_VERSION=v2.23.4-1
```

Comment:
hi @kwen2501 I believe here need to pin ``NCCL_VERSION_CU118=v2.21.5-1``
and use it in ``function install_118``

Reply:
I was just wondering about it :) Thanks!
I followed the `CUDNN_VERSION` practice and put an overwrite in `function install_118` now.